ui/src/app/utils/index.tsx

```diff
@@ -36,23 +36,7 @@ export function isBlank(toTest: String): boolean {
   }
 }
 
-/**
- * Random String Generator (simplified version)
- *
- * Info:      http://stackoverflow.com/a/27872144/383904
- * Use:       randomString(length);
- * Default:   return a random alpha-numeric string
- */
-export function randomString(len): string {
-  let str = '',
-    i = 0;
-  for (; i++ < len; ) {
-    let rand = Math.floor(Math.random() * 62);
-    const charCode = (rand += rand > 9 ? (rand < 36 ? 55 : 61) : 48);
-    str += String.fromCharCode(charCode);
-  }
-  return str;
-}
+export const isEmpty = (o: Object): boolean => Object.keys(o).length === 0;
```

Comment:
added this new check as a replacement for `fp.isEmpty()` - `fp` is now EOL so we should be replacing it wherever we can